cmd/ncdu/ncdu.go

```diff
@@ -911,6 +912,27 @@ func (u *UI) Show() error {
 	if err != nil {
 		return fmt.Errorf("screen init: %w", err)
 	}
+
+	// Hijack fs.LogPrint so that it doesn't corrupt the screen.
+	if logPrint := fs.LogPrint; !log.Redirected() {
+		type log struct {
+			text  string
+			level fs.LogLevel
+		}
+		var logs []log
+		fs.LogPrint = func(level fs.LogLevel, text string) {
+			if len(logs) > 100 {
+				logs = logs[len(logs)-100:]
+			}
+			logs = append(logs, log{level: level, text: text})
+		}
+		defer func() {
+			for i := range logs {
```

Comment:
This function should also replace `fs.LogPrint = logPrint` I think?

Reply:
True. I thought I pushed that, but it turns out I missed it.